experiments/tf_trainer/tf_hub_classifier/model.py

```diff
@@ -0,0 +1,83 @@
+"""Tensorflow Estimator implementation of RNN Model with Attention"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import tensorflow_hub as hub
+from tf_trainer.common import base_model
+from typing import Set
+
+FLAGS = tf.app.flags.FLAGS
+
+# Hyperparameters
+# TODO: Add validation
+tf.app.flags.DEFINE_float('learning_rate', 0.00003,
+                          'The learning rate to use during training.')
+tf.app.flags.DEFINE_float('dropout_rate', 0.3,
+                          'The dropout rate to use during training.')
+tf.app.flags.DEFINE_string(
+    'model_spec',
+    'https://tfhub.dev/google/universal-sentence-encoder/2',
+    'The url of the TF Hub sentence encoding module to use.')
+# This would normally just be a multi_integer, but we use string due to
+# constraints with ML Engine hyperparameter tuning.
+tf.app.flags.DEFINE_string(
+    'dense_units', '512,128,64',
+    'Comma delimited string for the number of hidden units in the dense layer.')
+
+
+class TFRNNModel(base_model.BaseModel):
```

Comment:
Lets correct the class name. Maybe TFHubClassifierModel

Reply:
Done.